config/default.js

```diff
@@ -123,6 +123,7 @@ module.exports = {
     // if false, autopilot selection won't be available
     mainnetAutopilot: false,
     networkSelection: false,
+    scbRestore: true,
```

Comment:
I think the default should be `false` here

Reply:
updated